src/clzones.cpp

```diff
@@ -236,6 +289,15 @@ std::string plot_options::get_zone_name_suggestion() const
     return _( "No seed" );
 }
 
+std::vector<std::pair<std::string, std::string>> blueprint_options::get_descriptions() const
+{
+    std::vector<std::pair<std::string, std::string>> options = std::vector<std::pair<std::string, std::string>>();
```

Comment:
```suggestion
    std::vector<std::pair<std::string, std::string>> options =
                std::vector<std::pair<std::string, std::string>>();
```

Reply:
there's no need to call the constructor like that.  Should just be
```cpp
std::vector<std::pair<std::string, std::string>> options;
```